styles/themes/base/components/_button.scss

```diff
@@ -0,0 +1,47 @@
+// Theme
+// disabled:                 'disabled',
+// loading:                  'loading',
+// wrapper:                  'ld__button-wrapper',
+
+.ld__btn {
+  display: inline-block;
+  color: var(--btn-text-color);
+  background-color: var(--btn-bg-color);
+  border-radius: var(--border-radius);
+  border: 2px solid var(--btn-border-color);
+  font-size: var(--font-size);
+  font-weight: var(--font-weight);
+  padding-top: var(--btn-padding-top);
```

Comment:
Since you are rewriting classes anyway, don't you want to start using logical properties?
padding-block + padding-inline

Reply:
not sure it is needed here, you may want different x/y ratio in different projects

Reply:
I'm not following) padding-block converts to padding-top and padding-bottom, padding-inline converts to padding-left and padding-right.
It's just a different way to describe paddings based on the language direction, it does not limit you in any way

Reply:
so, just finished reading about logical properties.
we don't need them
but it's okay if we have them
it's nice to be able to replace padding-left and padding-right with one nice padding-inline, as tailwind does with px and py, but replacing everything would be too much
the aim of logical properties is to support multi-language markup
we don't have it neither we are going to have it in the nearest future
even if we have it, we need to understand how should we change components first, just using logical properties now will not do this job in advance, so it is a separate project and let's get to it as we are ready
anyways, thanks for sharing

Reply:
okay, less code may be better than more code